Valghalla.External.Application/Modules/MyProfile/Commands/UpdateMyProfileCommand.cs

```diff
@@ -25,7 +25,9 @@ public UpdateMyProfileCommandValidator()
 
             When(x => !string.IsNullOrEmpty(x.MobileNumber), () =>
             {
-                RuleFor(x => x.MobileNumber).Length(Constants.Validation.MobileNumberLength);
+                RuleFor(x => x.MobileNumber)
+                .Length(Constants.Validation.MobileNumberLength)
+                .Matches("^[0-9]*$").WithMessage("'Mobile Number' must contain only numbers.");
```

Comment:
The text should be localized, since it should show Danish in production

Reply:
It has been fixed